docs/serving/samples/cloudevents/cloudevents-rust/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name = "knative-cloudevents-example"
+version = "0.1.0"
+authors = ["Francesco Guardiani <francescoguard@gmail.com>"]
+edition = "2018"
```

Comment:
2018?

Reply:
it's the name of the rust edition :smile: in 2018 they decided to do this "edition" thing in order to do some major breaking changes https://blog.rust-lang.org/2018/07/27/what-is-rust-2018.html

Reply:
In fact I'm just using the latest version of rust